WalletWasabi.Backend/Controllers/BatchController.cs

```diff
@@ -36,7 +36,7 @@ public BatchController(BlockchainController blockchainController, ChaumianCoinJo
 		public OffchainController OffchainController { get; }
 
 		[HttpGet("synchronize")]
-		public async Task<IActionResult> GetSynchronizeAsync([FromQuery, Required]string bestKnownBlockHash, [FromQuery, Required]int maxNumberOfFilters, [FromQuery]string estimateSmartFeeMode)
+		public async Task<IActionResult> GetSynchronizeAsync([FromQuery, Required] string bestKnownBlockHash, [FromQuery, Required] int maxNumberOfFilters, [FromQuery] string? estimateSmartFeeMode)
```

Comment:
I would prefer to avoid null types whenever possible. In this situation the default value is well known: "CONSERVATIVE" what it is also explicit and let other to understand  which fee mode is used when it is not explicitly passed. 

Additionally, this practice removes ambiguity because the caller is the one who decides what has to be done, otherwise the responsibility is delegated to the callee and that can be a source of surprises. For example, in many RPC calls involving fee estimation modes Bitcoin Core assumes CONSERVATIVE while Bitcoin Knots assume UNSET (that I don't know what means and didn't went further neither).

Reply:
I agree with you, but

"If a community can't look at something with a -1/+1 LoC diff and say "yea, good enough, there's some other things we could add later, but I'm not willing to do the work, so just go with what you said", then that's a problem. But, indeed, maybe that's just the folks on the ML." - @TheBlueMatt

Reply:
Your idea is good Lucas but it cannot be done. The reasoning here: https://github.com/zkSNACKs/WalletWasabi/pull/4228#issuecomment-679960103

If you want to have an optional parameter this is the only way to go, so modelstate isvalid will be true even when the feemode is missing from the query.

`[FromQuery] string? estimateSmartFeeMode`